Move snap message (#1294)
MetaMask/metamask-docs: snaps/features/data-storage.md

100 MB of data to the user's disk and retrieve it at will.
We recommend using this method for storing data in a Snap long term.

:::caution important

> You can install Snaps into each user's MetaMask instance.

Reviewer: ```suggestion
Snaps are installed in each user's MetaMask instance.
```

The user installs the Snap, not the Snap developer. I think passive voice is fine here because we are emphasizing where the Snap is installed.
